### PR TITLE
Cleanup frontend dependencies and update Kubernetes dependencies

### PR DESCRIPTION
**Pull request by maciaszczykm**
Closes https://github.com/kubernetes/dashboard/issues/6634.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,7 +82,7 @@
     "sockjs-client": "1.5.2",
     "strip-ansi": "6.0.1",
     "systemjs": "6.11.0",
-    "truncate-url": "1.0.0",
+    "truncate-url": "2.1.0",
```
Comment (shu-mutou) [inline package.json:85]: The latest `truncate-url` seems to have breakable change for dashboard.

File: package.json [hunk 2]
```diff
@@ -94,22 +92,21 @@
   "devDependencies": {
     "@angular-devkit/build-angular": "13.1.2",
     "@angular/cli": "13.1.2",
```
Comment (shu-mutou) [inline package.json:94]: The latest version for angular/cli is 13.1.3. Recentry, a few angular staffs have version inconsistency.
Could you check with `npx npm update`?

**Comment by shu-mutou** [comment]
I created #6690 as a continuation of yesterday.
I ran the checks locally for each commit, so it probably works.
Now, it's the end of the day, so I'm leaving without seeing the CI results.
I'm preciated you would concentrate to remove unused packages. Thanks!

**Comment by maciaszczykm** [comment]
@shu-mutou @floreks PTAL

**Comment by maciaszczykm** [comment]
I've also updated k8s dependencies.

**Pull request merged**